.github/workflows/build-master.yml

```diff
@@ -36,7 +32,12 @@ jobs:
         aws-region: us-west-2
     - name: Download Test Resources
       run: |
+        pip install quilt3~=3.1.12
```

Comment:
does this step need the `pip install --upgrade pip` also? (compare with below two lines down)

Reply:
Nah, I only do that for the actual deps because I want to make sure that everything installs and tests correctly on always the latest pip and etc. This is just utlities so no real need to be "always up to date"